ecl/regress/issue24245a.ecl

```diff
@@ -0,0 +1,41 @@
+/*##############################################################################
```

Comment:
Deliberately not part of the runtime regression suite, but  used as test cases to ensure the problems are caught.
issue24245a catches the case where the allocation once it is rounded up overflows.  issue24245 catches the problem where the requested size will take it over 2^32.